cmd/soroban-cli/src/commands/contract/install.rs

```diff
@@ -153,6 +163,7 @@ impl NetworkRunnable for Cmd {
                             // Skip reupload if this isn't V0 because V1 extension already
                             // exists.
                             if code.ext.ne(&ContractCodeEntryExt::V0) {
+                                output.info("Reusing contract previously uploaded");
```

Comment:
The term reusing and contracts make it a little ambiugous as to what is happening. It makes it sound like the contract deployment is being reused and like maybe a deploy isn't taking place. When in fact we're just skipping the code install. Suggest changing to using the 'install' terminology to use consistent vocab and narrative:
```suggestion
                                output.info("Skipping install because wasm already installed");
```